covid_data_getter.py

```diff
@@ -14,6 +14,12 @@
 # Sets the name of the CSV file we'll be using to store data 
 file = 'ontario_covid_data.csv'
 
+# Gets today's date and formats it 
+today = date.today()
+ftoday = str(today)
+# Determines how many days back to check for data in the CSV file
+datesback = 10
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/#constants) this should be uppercase to denote it as a constant
```
DATESBACK = 10
```